### PR TITLE
Fix GPU benchmarks on PyTorch model tests.

### PR DESCRIPTION
**Pull request by monorimet**
Run:
```
CUDA_VERSION=116 IMPORTER=1 ./setup_venv.sh
```
```
pytest --benchmark tank/pytorch/bert-base-uncased/bert-base-uncased_test.py
```
```
vim bench_results.csv
```

### CODE REVIEW

File: setup_venv.sh [hunk 1]
```diff
@@ -108,6 +108,20 @@ fi
 
 $PYTHON -m pip install -e . --extra-index-url https://download.pytorch.org/whl/nightly/cpu -f https://github.com/llvm/torch-mlir/releases -f https://github.com/${RUNTIME}/releases
 
+if [[ ! -z "${CUDA_BENCHMARKS}" ]]; then
+  echo "${Yellow}Installing GPU benchmarking tools..."
+  if [[ $(uname -s) = 'Linux' ]]; then
+    echo "${Yellow}Linux detected.. installing Linux benchmarking tools"
+    $PYTHON -m pip uninstall -y torch torchvision
+    $PYTHON -m pip install --pre torch torchvision --extra-index-url https://download.pytorch.org/whl/nightly/cu116
```
Comment (powderluv) [inline setup_venv.sh:116]: can we always just install the cuda version on linux and not key off CUDA_BENCHMARKS ?

Comment (monorimet) [reply setup_venv.sh]: Yeah, I'll just add it under IMPORTER=1 which is required for TF benchmarks anyways.

Comment (monorimet) [reply setup_venv.sh]: I removed the CUDA_BENCHMARKS switch. Now, if CUDA_VERSION is set, PyTorch+CUDA will be installed for CUDA_VERSION.

Comment (monorimet) [reply setup_venv.sh]: Removed CUDA_VERSION.
torch+cuda116 will install for linux builds by default.

**Review by powderluv — APPROVED** [review]

**Pull request merged**